market/listeners.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'chris'
```

Comment:
That's just autogenerated and I forget to change it. I can go back and change them later.